app/models/forms/detainee.rb

```diff
@@ -26,6 +26,10 @@ def validate_date_of_birth
       errors.add(:date_of_birth) unless date_of_birth.is_a? Date
     end
 
+    def prison_number=(value)
+      value && super(value.upcase)
```

Comment:
Seems as this in a few places now prison number could become a value object. Something like:

```ruby
class PrisonNumber < SimpleDelegator
  def initialize(value)
    super(value&.upcase)
  end
end
```